mythril/laser/ethereum/instructions.py

```diff
@@ -1302,7 +1302,7 @@ def call_(self, global_state: GlobalState) -> List[GlobalState]:
 
         if 0 < int(callee_address, 16) < 5:
             log.debug("Native contract called: " + callee_address)
-            if call_data == [] and call_data_type == CalldataType.SYMBOLIC:
+            if call_data == [] and isinstance(call_data, SymbolicCalldata):
```

Comment:
This whole if statement should be removed. `call_data` will never be an empty list.

Reply:
Cool

Reply:
Thanks for removing :laughing: